dali/pipeline/data/types.h

```diff
@@ -182,6 +182,13 @@ class DLL_PUBLIC TypeInfo {
   string name_;
 };
 
+template <typename T>
+struct DLL_PUBLIC TypeNameHelper {
```

Comment:
Do those helpers need to be DLL_PUBLIC and public after all?

Reply:
You're right, I forgot to remove this. Done.